x/evidence/client/rest/query.go

```diff
@@ -22,6 +22,11 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 		"/evidence",
 		queryAllEvidenceHandler(cliCtx),
 	).Methods(MethodGet)
+
+	r.HandleFunc(
+		"/evidence/params",
+		queryParamsHandler(cliCtx),
+	).Methods("GET")
```

Comment:
Should this be changed to:

```suggestion
	).Methods(MethodGet)
```